base_multi_image/models/image.py

```diff
@@ -26,11 +27,8 @@ class Image(models.Model):
         ),
     ]
 
-    owner_id = fields.Integer(
-        "Owner",
-        required=True,
-        ondelete="cascade",  # This Integer is really a split Many2one
-    )
+    # This Integer is really a split Many2one
+    owner_id = fields.Integer("Owner", required=True)
```

Comment:
@hailangvn Why removing the ondelete ?

Reply:
Good day @rousseldenis, thanks for the question. Odoo had warning about `ondelete` because the `owner_id` is `fields.Integer`, not `fields.Many2one`.
```
WARNING odoo.fields: Field base_multi_image.image.owner_id: unknown parameter 'ondelete', if this is an actual parameter you may want to override the method _valid_field_parameter on the relevant model in order to allow it
```